packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -406,69 +406,89 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 { eventName: "ObtainSnapshot" },
                 async (event: PerformanceEvent) => {
                     const props = {};
-                    let cachedSnapshot: ISnapshotContents | undefined;
+                    let retrievedSnapshot: ISnapshotContents | undefined;
+                    // Here's the logic to grab the persistent cache snapshot implemented by the host
+                    // Epoch tracker is responsible for communicating with the persistent cache, handling epochs and cache versions
                     const cachedSnapshotP: Promise<ISnapshotContents | undefined> =
                         this.epochTracker.get(createCacheSnapshotKey(this.odspResolvedUrl))
-                            .then((snapshotCachedEntry: ISnapshotCachedEntry) => {
+                            .then(async (snapshotCachedEntry: ISnapshotCachedEntry) => {
                                 if (snapshotCachedEntry !== undefined) {
                                     // If the cached entry does not contain the entry time, then assign it a default of 30 days old.
-                                    // eslint-disable-next-line @typescript-eslint/dot-notation
-                                    props["cacheEntryAge"] = Date.now() - (snapshotCachedEntry.cacheEntryTime ??
+                                    const age = Date.now() - (snapshotCachedEntry.cacheEntryTime ??
                                         (Date.now() - 30 * 24 * 60 * 60 * 1000));
+
+                                    // Record the cache age
+                                    // eslint-disable-next-line @typescript-eslint/dot-notation
+                                    props["cacheEntryAge"] = age;
+
+                                    // TODO: Refactor this logic (maybe move to a separate function)
+                                    // TODO: Currently there is a max of two days. Potentially remove this logic once proactive expiry exists.
+                                    // Set the max age to 2 days, a network call to retrieve the snapshot will be made if undefined is returned.
+                                    const twoDays = 2 * 24 * 60 * 60 * 1000;
+                                    const maxCacheAge = twoDays;
```

Comment:
Make `twoDays` a global const in this file. And probably call it something like `defaultCacheExpiryTimeoutMs` or something like that. That would make it very explicit that this is the default.
The `Ms` at the end tells the unit of time which can be helpful. You should add this to other variable names that are dealing with time in ms.

Reply:
You should also think about exporting this global default to use in tests.